### PR TITLE
Dynamic Scene Output

### PR DESCRIPTION
**Pull request by alexyao2015**
Debug #231 here

### CODE REVIEW

**Comment by alexyao2015** [comment]
@mariusmotea 
Lets try to figure out what went wrong. I've been running my fork with this change for at least a week now and its fine.

Also lets use "squash and merge" for future pull requests so things are easier to see in commit history.

File: BridgeEmulator/HueEmulator3.py [hunk 1]
```diff
@@ -1412,9 +1424,29 @@ def do_GET(self):
...
-                    self._set_end_headers(bytes(json.dumps({"lights": bridge_config["lights"], "groups": bridge_config["groups"], "config": bridge_config["config"], "scenes": bridge_config["scenes"], "schedules": bridge_config["schedules"], "rules": bridge_config["rules"], "sensors": bridge_config["sensors"], "resourcelinks": bridge_config["resourcelinks"]},separators=(',', ':'),ensure_ascii=False), "utf8"))
+                    #trim off lightstates as per hue api
+                    scenelist = {}
+                    for scene in bridge_config["scenes"]:
+                        scenelist["scenes"][scene] = bridge_config["scenes"][scene]
+                        if "lightstates" in scenelist["scenes"][scene]:
+                            del scenelist["scenes"][scene]["lightstates"]
+                        if ("type" in scenelist["scenes"][scene]) and ("GroupScene" == scenelist["scenes"][scene]["type"]):
+                            scenelist["scenes"][scene]["lights"] = {}
+                            scenelist["scenes"][scene]["lights"] = scenelist["groups"][scenelist["scenes"][scene]["group"]]["lights"]
```
Comment (Thomas-Vos) [inline BridgeEmulator/HueEmulator3.py]: What exactly does this line do? What is `scenelist["groups"]`?

**Comment by alexyao2015** [comment]
Alright I should have fixed that error you were seeing @mariusmotea

**Comment by alexyao2015** [comment]
Any objections to me merging this? @mariusmotea

**Comment by mariusmotea** [comment]
I thing is more easy just to call the sanitizeBridgeScenes function en every HTTP PUT request where url path 2 is `groups`. I saw the code put now in webserver PUT request handling is just a copy of the function content so is the same if you call the function.

**Comment by alexyao2015** [comment]
I'll take care of the conflicts and merge it tonight

**Pull request merged**